spec/support/vcr_cassettes/vaos/concerns/token/token_400.yml

```diff
@@ -0,0 +1,31 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://test.example.com/token
```

Comment:
Please add this value to the VCR helper and hide the URI.

Reply:
Fixed!